src/main/java/FileFinder.java

```diff
@@ -0,0 +1,52 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+class FileFinder implements ResourceFinder {
+    private String rootPath;
+
+    public FileFinder(String rootPath) {
+        this.rootPath = rootPath;
+    }
+
+    public String getContentOf(String resourcePath) {
+        try {
+            System.out.println("Looking up resource at location: " + rootPath + resourcePath);
```

Comment:
Avoid logging information to the console.

You most likely had this here to see what was going on (spiking?) and just forgot to remove it. Logging to the console is very helpful for finding little wrinkles in what we do, but they should not survive after we found the knowledge we were seeking.